ontologies/Items.ttl

```diff
@@ -0,0 +1,171 @@
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+  
+wi:hasLocation rdf:type owl:ObjectProperty ;
+	rdfs:label "hasLocation"@en ;
+	rdfs:comment "This relates an item to a broad division of the Library or its storage areas within which an item is held: for instance, the History of Medicine collection, or the Closed Store Archives and Manuscripts material."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range wi:Classmark ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+	
+wi:hasFileFormat rdf:type owl:Class ;
+	rdfs:label "hasFileFormat"@en ;
+	rdfs:comment "This relates an item to its file format when the item comprises a digital object."@en ;
+ 	rdfs:domain wi:Item ;
+	rdfs:range wi:FileFormat ;
+  rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+  
+  
+#######  Data properties #####  
+	
+wi:barcode rdf:type owl:DatatypeProperty ;
+	rdfs:label "barcode"@en ;
+	rdfs:comment "Relates the item to a number string encoded on physical items by a barcode and used for the purposes of movement control."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range rdf:langString ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+  
+wi:boxNumber rdf:type owl:DatatypeProperty ;
+	rdfs:label "boxNumber"@en ;
+	rdfs:comment "Relates the item (generally an archive item) to a number identifying the box within which it is stored."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range rdf:langString ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+  
+wi:outsizeItemNumber rdf:type owl:DatatypeProperty ;
+	rdfs:label "boxNumber"@en ;
+	rdfs:comment "Relates the item (generally an archive item) to a number identifying its storage location when the item is too large to fit into a standard Library box."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range rdf:langString ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .	
+  
+wi:physicalDescription rdf:type owl:DatatypeProperty ;
+	rdfs:label "physicalDescription"@en ;
+	rdfs:comment "A description of specific physical characteristics of the item - condition, dimensions, details of cover or binding, etc. - or, in the case of digital objects, any notes relating to this digital manifestation over and above notes of file format."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range rdf:langString ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+  
+wi:sheetSize rdf:type owl:DatatypeProperty ;
+	rdfs:label "sheetSize"@en ;
+	rdfs:comment "A note of the width and length dimensions of a flat artwork."@en ;
+	rdfs:domain wi:Item ;
+	rdfs:range rdf:langString ;
+	rdf:isDefinedBy <http://www.wellcomecollection.org/ontologies/items> .
+   
+wi:acquisitionDetails rdf:type owl:DatatypeProperty ;
+	rdfs:label "acquisitionDetails"@en ;
+	rdfs:comment "A note of how this item was acquired by the Wellcome Library: source, price if one was paid, date of acquisition, and any other pertinent details."@en ;
```

Comment:
Fair enough - and we talked about that stylistic issue elsewhere across Data Properties, too.